src/support/mod.rs

```diff
@@ -137,12 +138,49 @@ pub fn get_llvm_version() -> (u32, u32, u32) {
     return (major, minor, patch);
 }
 
+/// Permanently load the dynamic library with the given `filename`.
+///
+/// It is safe to call this function multiple times for the same library.
+///
+/// Returns `true` if there was an error while loading the library.
+#[deprecated(note = "Use `try_load_library_permanently` instead")]
```

Comment:
Could you please add that it will be removed in a future version?

Reply:
I'd suggest to have an update cycle `load_library_permanently` -> `try_load_library_permanently` -> `load_library_permanently`, instead of removing it.

In the end, `load_library_permanently` should have the same signature as `try_load_library_permanently`

Reply:
We should maybe just change it then, I'm not really concerned about breaking changes since we're pre-1.0